core/src/trezor/ui/confirm.py

```diff
@@ -10,6 +10,8 @@
 )
 from trezor.ui.loader import Loader, LoaderDefault
 
+from .model import confirm as model
```

Comment:
thought (not necessarily for this PR):
the usage `from .model import <name> as model` suggests a different possibility:
```
def import_model_specific(name: str) -> Module:
	model_name = "t1" if is_t1 else "tt"
	widget_name = name.split(".")[-1]
	return __import__("trezor.ui.model.{}.{}".format(widget_name, model_name), None, None, (widget_name,), 0)

model = import_model_specific(__name__)
```

This would avoid the need to have any code at all in `model/*/__init__.py`. It would preclude the thing where the build system replaces the `__init__` with the appropriate module directly, but that might not matter.

what do you all think?

Reply:
another drawback is that this would probably be untypeable...